webhook_to_fedora_messaging/move.py

```diff
@@ -0,0 +1,145 @@
+
+
+def read_user():
+    with open(userscsv_fileloca) as file:
+        useriter = csv.reader(file)
+        next(useriter)
+        for item in useriter:
+            yield item[0]
+
+
+def read_repo():
+    with open(reposcsv_fileloca) as file:
+        repoiter = csv.reader(file)
+        next(repoiter)
+        for item in repoiter:
+            yield item[1], item[2], item[3], item[4], item[5]
+
+
+def read_pair():
+    pair, userdict = dict(), dict()
+    with open(userscsv_fileloca) as file:
+        useriter = csv.reader(file)
+        next(useriter)
+        for item in useriter:
+            userdict[item[1]] = item[0]
+    with open(pairscsv_fileloca) as file:
+        pairiter = csv.reader(file)
+        next(pairiter)
+        for item in pairiter:
+            if item[1] not in pair:
+                pair[item[1]] = [userdict[item[0]]]
+            else:
+                pair[item[1]].append(userdict[item[0]])
+    print(dumps(pair, sort_keys=True, indent=2))
+    return pair
+
+
+async def import_user_to_database():
+    async for sess in get_session():
+        for item in read_user():
+            try:
+                if "github_org" not in item:
+                    make_user = User(name=item)
+                    sess.add(make_user)
+                    await sess.flush()
+                    await sess.commit()
+                    print(f"[{datetime.now().strftime(format)}] User '{item}' created.")
+                else:
+                    print(f"[{datetime.now().strftime(format)}] User '{item}' abandoned.")
+            except Exception as expt:
+                await sess.rollback()
+                print(f"[{datetime.now().strftime(format)}] User '{item}' failed.")
+                print(f"An error occurred: {expt}")
+            finally:
+                await sess.close()
+
+
+async def import_repo_to_database():
+    pairdata, done = read_pair(), set()
+    async for sess in get_session():
+        for item in read_repo():
+            name, desc, lang, team, active = item
+            try:
+                flag = 0
+                if team in pairdata:
+                    flag = 1
+                else:
+                    try:
+                        flag = 2
+                        qery = select(User).filter_by(name=team).options(selectinload("*"))
+                        rslt = await sess.execute(qery)
+                        user = rslt.scalar_one()
+                    except NoResultFound:
+                        print(f"[{datetime.now().strftime(format)}] Service '{team}/{name}' abandoned.")
+                        continue
+
+                # SERVICE IS TO BE CREATED ONLY IF FLAG IS 1 OR 2
+                # ORPHANS ARE TO BE ABANDONED
+                make_service = Service(
+                    name=f"{team}/{name}",
+                    uuid=uuid4().hex[0:8],
+                    type="github",
+                    desc=f"Written in {lang}. {desc}" if lang != "None" else f"{desc}",
```

Comment:
`f"{desc}" == desc`, no?

Reply:
A bunch of the repository entries have languages associated with them. I am trying to use that column instead of simply disposing of it. I would be interested to know if I should not do it.

Reply:
I'm not sure what we should do with this column, actually. Maybe add "Imported from github2fedmsg"? What did you think it should be eventually, for services created by users? The repo summary as seen on the top right of the github UI? Something else?